### DISCUSSION TITLE
QR for builds

### DISCUSSION THREAD

**Discussion by yairpalti** [codemagic-ci-cd/codemagic-docs#2322] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Till last week I have an option to send QR as direct link to the build artifact to others.
Since last week, there is no QR option near the APK or the IPA and I have no way to send a download link

### Expected results

QR for download link

### Actual results

No direct link for download

**Comment by dtrdic** [comment]
Hello @yairpalti if you are using the workflow editor, please choose the third option from the dropdown to view the QR code download link when the build is successful
<img width="949" alt="Screenshot 2024-01-19 at 10 10 05" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/446f08cf-95bd-49ec-bd42-0f7f8cad38f1">

But please share a build id so I can take a look. You can find it in the build overview page.

**Reply by yairpalti** [reply]
<img width="466" alt="Screenshot 2024-01-19 at 12 47 30" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/23010207/60a78b70-9aa0-44d2-85c5-cf3679916e32">

As you can see the snapshot, I don't have the QR and even the link isn't open for share

https://codemagic.io/app/656ee95af946f8c9852782b1/build/65a93e8bd0ecff443c0f4f61

**Reply by dtrdic** [reply]
This is because the availability of certain features, like QR codes, is tied to team-specific account but you are using a personal account. To access these feature, consider creating a team (Teams is a paid feature on Codemagic) see our [pricing page](https://codemagic.io/pricing/)

**Comment by dtrdic** [comment]
I am closing the thread due to inactivity. Let me know if you have any more questions by replying to this thread or opening a new one.
